src/replaykit.cs

```diff
@@ -203,9 +204,12 @@ interface RPBroadcastConfiguration : NSCoding, NSSecureCoding {
 		[Export ("clipDuration")]
 		double ClipDuration { get; set; }
 
-		// TODO review, once API documentation is available, if we can provide an easier to use strongdictionary
 		[NullAllowed, Export ("videoCompressionProperties", ArgumentSemantic.Strong)]
-		NSDictionary<NSNumber, INSSecureCoding> VideoCompressionProperties { get; set; }
+#if XAMCORE_2_0
```

Comment:
ouch, but that's now a breaking change :( somehow it did not show (or I did not see) it while grepping for TODO in my diffs

Add an [Obsolete] on the existing API and I think you can remove the XAMCORE_2_0 (unless it's on the mac too).

The rest (strongdict) is good to go (once the branching is complete for the release).